app/models/concerns/stacks_images.rb

```diff
@@ -0,0 +1,25 @@
+###
+#  Mixin to add methods for crafting Stacks image urls
+###
+module StacksImages
+  ##
+  # Generates a Stacks image url using given arguments
+  # @param [String] druid
+  # @param [String] image_id
+  # @param [Symbol] size
+  # @return [String]
+  def craft_image_url(druid, image_id, size)
```

Comment:
We should update this to use IIIF sometime. If you visit the generated urls, it'll redirect you to the IIIF equivalents.